bftengine/src/bftengine/ClientsManager.cpp

```diff
@@ -104,6 +104,8 @@ void ClientsManager::init(IStateTransfer* stateTransfer) {
 
 uint32_t ClientsManager::numberOfRequiredReservedPages() const { return requiredNumberOfPages_; }
 
+std::set<NodeIdType> ClientsManager::getInvalidClients() { return invalidClients; }
```

Comment:
We don't need to keep track of invalid clients.  In fact, this might actually be an attack vector, as we would grow memory for each invalid client and we never clear it. We only need a global counter like for noops below. Then we will be able to monitor it for changes.

Reply:
Yeah, I added a set to have the count unique. But yeah, agree that it could be dangerous in the long run, will keep in mind of such aspects for future PRs. Thanks. 

Changed it to using a numerical counter